### ISSUE TITLE
Dependencies problem

### ISSUE THREAD

**Issue by ThomasBouet** [bluejeans/web-client-sdk#1]
After following the given procedure in the ReadMe :
`npm i @bluejeans/web-client-sdk`
I have the following problems when I build my application in angular
`Error: node_modules/@bluejeans/web-client-sdk/dist/types/logger.d.ts:1:28 - error TS2307: Cannot find module '@bluejeans/bjn-core-webrtc' or its corresponding type declarations.

1 import { LoggerIntf } from '@bluejeans/bjn-core-webrtc';
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/@bluejeans/web-client-sdk/dist/types/services/ParticipantService.d.ts:2:48 - error TS2307: Cannot find module '@bluejeans/bjn-core-webrtc' or its corresponding type declarations.

2 import { Participant as CoreParticipant } from "@bluejeans/bjn-core-webrtc";
                                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Error: node_modules/@bluejeans/web-client-sdk/dist/types/services/PrivateChatService.d.ts:1:31 - error TS2307: Cannot find module 'subscribableevent' or its corresponding type declarations.

1 import SubscribableEvent from 'subscribableevent';
                                ~~~~~~~~~~~~~~~~~~~


Error: node_modules/@bluejeans/web-client-sdk/dist/types/services/PublicChatService.d.ts:1:31 - error TS2307: Cannot find module 'subscribableevent' or its corresponding type declarations.

1 import SubscribableEvent from 'subscribableevent';
                                ~~~~~~~~~~~~~~~~~~~

Error: node_modules/@bluejeans/web-client-sdk/dist/types/utils/Environments.d.ts:1:28 - error TS2307: Cannot find module '@bluejeans/bjn-core-webrtc' or its corresponding type declarations.

1 import { ClientInfo } from "@bluejeans/bjn-core-webrtc";
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~




** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **


✖ Failed to compile.
✔ Browser application bundle generation complete.

5 unchanged chunks

Build at: 2022-02-28T16:04:23.000Z - Hash: 316221e4617843645bb6 - Time: 1025ms

Error: node_modules/@bluejeans/web-client-sdk/dist/types/logger.d.ts:1:28 - error TS2307: Cannot find module '@bluejeans/bjn-core-webrtc' or its corresponding type declarations.

1 import { LoggerIntf } from '@bluejeans/bjn-core-webrtc';
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Error: node_modules/@bluejeans/web-client-sdk/dist/types/services/ParticipantService.d.ts:2:48 - error TS2307: Cannot find module '@bluejeans/bjn-core-webrtc' or its corresponding type declarations.

2 import { Participant as CoreParticipant } from "@bluejeans/bjn-core-webrtc";
                                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/@bluejeans/web-client-sdk/dist/types/services/PrivateChatService.d.ts:1:31 - error TS2307: Cannot find module 'subscribableevent' or its corresponding type declarations.

1 import SubscribableEvent from 'subscribableevent';
                                ~~~~~~~~~~~~~~~~~~~


Error: node_modules/@bluejeans/web-client-sdk/dist/types/services/PublicChatService.d.ts:1:31 - error TS2307: Cannot find module 'subscribableevent' or its corresponding type declarations.

1 import SubscribableEvent from 'subscribableevent';
                                ~~~~~~~~~~~~~~~~~~~


Error: node_modules/@bluejeans/web-client-sdk/dist/types/utils/Environments.d.ts:1:28 - error TS2307: Cannot find module '@bluejeans/bjn-core-webrtc' or its corresponding type declarations.

1 import { ClientInfo } from "@bluejeans/bjn-core-webrtc";`

It seems that some necessary dependency are not installed.
I tried to install @bluejeans/bjn-core-webrtc but npm did not found it.

I use angular 12.2.0.

Thanks in advance for your answers

**Comment by YashGogia** [comment]
Hi @ThomasBouet,

We'll take a look and get back to you. Meanwhile, if you could share an example project (preferably something like [codesandbox](https://codesandbox.io/)) where we can reproduce the issue, that would help

**Comment by YashGogia** [comment]
This issue was caused because the type definition files of the web-client-sdk referring to some of the internal interfaces which were not exposed or defined. These issues have been addressed in the 1.4.0 release of the web-client-sdk. Furthermore, we have added checks to ensure that this doesn't happen again in the future
